HackBeanpot/internal-tools
# Parse CSV

In addition to the requirements scoped in the ticket, Dhivas and I added a table to render the csv file that is imported. 

- [ ] Created a new page emailSender.ts with header h3 titled 'Email sender", using Typography from MUI (like the other pages) 
- [ ] Created a button to upload / import a CSV file 
- [ ] Implemented functionality to upload and parse a CSV file 
- [ ] Prints out each message that will be emailed out 

**Note: currently does not print new lines -> will add this w/ other enhancements to a future sprint 

To test the ticket, you can use this message: 

```
Hi ${name}! My name is Cindy Luo, and I’m on HackBeanpot’s sponsorship team this year. I’m reaching out because we’ve started planning for HackBeanpot in February 2022, and we would love for ${company} to partner with us again! 

This past event was a success, with 200 students from 33 schools, creating 39 different projects, and it would not have been possible without you. Thanks to ${company}'s sponsorship, we were able to expose both new and returning hackers to the company's core values and mission statement! We plan to hold an in-person Hackathon but are open to remote sponsors, speakers, and mentors. We are closely monitoring public health guidance and will do our best to provide accommodations to make the sponsor experience as smooth as possible in light of COVID-19. I’ve attached our sponsorship packet to this email for you to look over, and would love to schedule a call to chat about it some more, and see what package might work best for ${company}. 

If you’re interested, please let me know when would be a good time. In the meantime, feel free to check out our website or Instagram to see what we’ve been up to! Thanks again for your continued support! We really appreciate it, and we are looking forward to hearing from you soon. 

Best, Cindy
```

and this CSV file [example.csv](https://github.com/HackBeanpot/internal-tools/files/8923724/example.csv)

DEMO: 
https://user-images.githubusercontent.com/59738880/174203646-cc883d3a-19cc-4e6f-b23b-18b398935157.mov
